src/registrar/models/waffle_flag.py

```diff
@@ -0,0 +1,69 @@
+from waffle.models import AbstractUserFlag
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class WaffleFlag(AbstractUserFlag):
+    """
+    Custom implementation of django-waffles 'Flag' object.
+    Read more here: https://waffle.readthedocs.io/en/stable/types/flag.html
+
+    Use this class when dealing with feature flags, such as profile_feature.
+    """
+
+    class Meta:
+        """Contains meta information about this class"""
+
+        verbose_name = "waffle flag"
+        verbose_name_plural = "Waffle flags"
+
+    @staticmethod
+    def get_default_waffle_flags():
+        """
+        Defines which waffle flags should be created at startup.
+
+        Add to this function if you want to add another flag that is generated at startup.
+        When you do so, you will need to add a new instance of `0091_create_waffle_flags_v{version_number}`
+        in registrar/migrations for that change to update automatically on migrate.
+        """
+        default_flags = [
```

Comment:
**blocking question:** Hmm why are we hard coding these? The purpose is for us to just use django admin and not have this at all except for where we are checking the flag values. Did you run into an issue with making them in admin etc? (this same comment applies for having a migration to preload with defaults